src/com/facebook/buck/jvm/kotlin/KotlinTestDescription.java

```diff
@@ -168,27 +164,5 @@ public void findDepsForTargetFromConstructorArgs(
   @BuckStyleImmutable
   @Value.Immutable
   interface AbstractKotlinTestDescriptionArg
-      extends HasContacts, HasTestTimeout, KotlinLibraryDescription.CoreArg {
-    ImmutableList<String> getVmArgs();
-
-    Optional<TestType> getTestType();
-
-    Optional<Level> getStdErrLogLevel();
-
-    Optional<Level> getStdOutLogLevel();
-
-    Optional<Long> getTestCaseTimeoutMs();
-
-    ImmutableMap<String, String> getEnv();
-
-    @Value.Default
-    default boolean getRunTestSeparately() {
-      return false;
-    }
-
-    @Value.Default
-    default ForkMode getForkMode() {
-      return ForkMode.NONE;
-    }
-  }
+      extends KotlinLibraryDescription.CoreArg, JavaTestDescription.CoreArg {}
```

Comment:
`KotlinLibraryDescription.CoreArg` already inherits from `JavaLibraryDescription.CoreArg`. So this change doe snot bring in anything more